embeddings/transformation/transformation.py

```diff
@@ -18,3 +18,14 @@ def __repr__(self) -> str:
     @abc.abstractmethod
     def transform(self, data: Input) -> Output:
         pass
+
+
+class Transformations(Transformation[Any, Any]):
+    def __init__(self, transformations: List[Transformation[Any, Any]]):
```

Comment:
nieeeee no wlasnie o to chodzi zeby miec transformacje bezpieczne, czyli nie ze lista z any do any, a klasa ktora bierze transformacje z a do b i z b do c i robi z tego transformacje z a do c, i w ten sposob mozna miec type safe transformacje, tak nie mamy tego

Reply:
Rozumiem całą koncepcję, to było tymczasowe rozwiązanie i też miałem dopytać. W takim razie jak tutaj wprowadzić taką listę transformacji bezpiecznie i czy powinno być dodatkowa walidacja czy typy się zgadzaja?